packages/core/src/config.ts

```diff
@@ -49,6 +49,7 @@ export interface MedplumSourceInfraConfig {
   rdsSecretsArn?: ValueOrExternalSecret<string>;
   rdsReaderInstanceType?: ValueOrExternalSecret<string>;
   rdsProxyEnabled?: ValueOrExternalSecret<boolean>;
+  rdsDBParameters?: StringMap;
```

Comment:
Any objections to simply `rdsParameters`?  Or maybe `rdsClusterParameters` to be more specific?

`DB` is a tricky abbreviation.

I suppose we could always go `rdsDatabaseParameters`?

Naming things is hard.